pegasus/sites.v3/code.org/views/mobile_footer_responsive.haml

```diff
@@ -1,11 +1,48 @@
--# an utra-simple mobile footer
+%link{:href=>'/css/mobile-footer.css', :rel=>'stylesheet'}
+%script{src: '/js/mobile-footer.js'}
 
 %footer#pagefooter
-  .content{:style=>'width:100%'}
-    .left{:style=>'float:left'}
-      - if request.language == "en"
-        %a.whitefooterlink{:href=>"/privacy", :style=>"color: #ffa400;"} Privacy Policy
-      %br
-      !="&copy; Code.org, #{Time.now.year}"
-    .right{:style=>'float:right'}
+  #footer-links
+    %ul
+      %li
+        %a{:href=>'/privacy', :style=>'color: #ffa400;'} Privacy Policy
+      %li
+        %a{:href=>'/about'} About
+      %li
+        %a{href:resolve_url('/partners')} Partners
```

Comment:
You don't need `resolve_url` in this file. This is only need on hourofcode.com to redirect to chapter partner sites like uk.code.org instead of code.org. You don't have to worry about chapter partner sites since they have their own separate footer.